gradle-plugin/src/main/kotlin/com/google/devtools/ksp/gradle/KspSubplugin.kt

```diff
@@ -410,8 +410,8 @@ class KspGradleSubplugin @Inject internal constructor(private val registry: Tool
         } else {
```

Comment:
This `else` block is KSP1 only. Does KSP2 (in the above `if` block) work out of the box, or could you consider making it work for KSP2 as well?

Reply:
I know almost nothing about KSP2. If you say it's also should be fixed - I will update CL to make it happen.
Assuming KSP2 can be switched on by some system propert